ref/README.md

```diff
@@ -1,56 +1,160 @@
+# The YPPM reference kernel
+
+NOTE: If you are reading this with a plain text editor, please note that this document is
+formatted with Markdown syntax elements.  See https://www.markdownguide.org/cheat-sheet/
+for more information.
+
 This is the reference implementation of the `yppm` kernel extracted from the FV3 model.
-The number of threads is determined by the OMP_NUM_THREADS environment variable, which
-defaults to the number of hyperthreads on the machine if it is not set.  Currently the
-number of threads is hard-coded to 4 in the build script when running "ctest", but users
-can customize it at runtime for other runs.
+It is an excellent kernel for OpenMP analysis. The number of threads is determined by
+the `OMP_NUM_THREADS` environment variable, which defaults to the number of hyperthreads
+on the machine if it is not set.  Currently the number of threads is hard-coded to 4 in
+the build script when running `ctest`, but users can customize it at runtime for other
+runs.
```

Comment:
Agree with Lynd.  I forgot that this kernel does not have any OpenMP in it.  So, the sentences about OpenMP analysis and OMP_NUM_THREADS should be replace with a short sentence that says something like:

This kernel does not contain any OpenMP threading.  Current analysis is limited to serial, single threaded performance.